packages/atomic/src/components/facets/atomic-color-facet/atomic-color-facet.tsx

```diff
@@ -296,9 +296,7 @@ export class AtomicColorFacet
       this.bindings.i18n
     );
     const isSelected = facetValue.state === 'selected';
-    const partValue = displayValue
-      .match(new RegExp('-?[_a-zA-Z]+[_a-zA-Z0-9-]*'))
-      ?.toString();
+    const partValue = facetValue.value.replace(/[^a-z0-9]/gi, '');
```

Comment:
This could technically be considered a breaking change, since it was using `displayValue` before, but now is using the actual value coming from the API.

It's just that it does not seem right to be using the display value here ?

You would need one css file for each language you want to support... 😅

Reply:
Will use 2 parts